include/picongpu/plugins/openPMD/openPMDWriter.x.cpp

```diff
@@ -1658,6 +1660,7 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
 
                     if(numDataPoints == 0)
                     {
+                        // avoid deadlock between not finished pmacc tasks and mpi blocking collectives
```

Comment:
@psychocoderHPC here you just added the comment, but not the code `eventSystem::getTransactionEvent().waitForFinished();`. Is this correct?

Reply:
no this is a bug :-(

Reply:
fixed